kitsune/gallery/forms.py

```diff
@@ -80,11 +78,6 @@ def __init__(self, *args, **kwargs):
         url = 'http://infohost.nmt.edu/tcc/help/pubs/pil/formats.html'
         self.fields['file'].help_text = msg.format(learn_more=url)
 
-    def clean(self):
```

Comment:
I think this should not be removed. We want only certain format file can be added. https://github.com/mozilla/kitsune/blob/master/kitsune/upload/forms.py#L34

Reply:
You're right, this is a change in behaviour. My reasoning was to rely on `django.core.validators:validate_image_file_extension`, but you may not want every single image format that pillow can open...

Note to self: This should be done with a custom image field which has `validators=[FileExtensionValidator(allowed_extensions=ALLOWED_IMAGE_EXTENSIONS)` rather than a clean method though.

Reply:
@SmileyChris Yeah. I understand. I think in that case you need to add this to `default_validators`

Reply:
Fixed in a0b7f0e7ca5c730d116a75d9403bb50f1082ecfd